src/modules/umount/README.md

```diff
@@ -0,0 +1,17 @@
+### Umount Module
+---------
+This module represents the last part of the installation, the unmounting of partitions used for the install.  It is also the last place where it is possible to copy files to the target system, thus the best place to copy an installation log.
+
+To be able to use the copy an installation log of this module you will need to include a log creation to your launcher script or add it to the used calamares.desktop, example of a launcher script:
+
+```
+#!/bin/sh
+sudo /usr/bin/calamares -d > installation.log 
+```
+Example desktop line:
+```
+Exec=sudo /usr/bin/calamares -d > installation.log
```

Comment:
We already write the log to disk, is this necessary? I've just pushed a change which makes Calamares print out the log path for convenience. This path is built to be in the user's cache directory, which under Plasma ends up as `~/.cache/Calamares/Calamares/Calamares.log`. It's easier and more reliable to just copy the log file from there.